### ISSUE TITLE
Question: how to redirect to a route after click next in step #3 for example?

### ISSUE THREAD

**Issue by lokize** [restimel/vue3-tutorial#1]
Hi, thanks for this project, i have one question:

how to redirect to a route after click next in step #3 for example?

**Comment by restimel** [comment]
Hello,

Do you mean something like a "goto step X"?

There is currently no built-in solution.
In the tutorial description, it is possible to move forward if `skipStep` returns `true`.

Something like
* step#1: point to a button
* step#2: skipStep if there are no error (`{ target: '.error', check: 'is not rendered',}`), point to the error message
* step#3: point to an image

So when clicking on "next" in step#1, if there are no error we move directly to step#3

It is planned to add a feature to go to step X instead of next and probably to add a props to force moving directly to a step.
I am not really sure how it should look like (should it be an option which indicates the next step index? the index increment? should we have id on step in order to point to correct step?)
Once it will be designed, I don't think it would be too hard to implement it.

Currently from my usage skipStep was enough so it was not really important for me to develop this feature. But it depends on your use case.
If you have one, I could use it as an example.

**Comment by lokize** [comment]
No, for example, I wanted to open a new route and continue the tutorial on this new route, for example initially the tutorial starts on the dashboard but it could follow another route starting from step 10

**Comment by restimel** [comment]
How will you do the choice between going to step2 or to step10?

Is something like this, match your case?
```
nextStep: (tutorial) => {
    if (something) {
        return 10;
    }
    return 2;
}
```

_I don't really like absolute index, because it seems very weak against changes, but this could be a base for the feature_

**Comment by restimel** [comment]
Hello,

I have added a feature which allow to choose to which step you want to navigate: previousStep and nextStep.
It accepts different format:
 * number: to navigate directly to the index of the step
 * string (format '+X' or '-X' where X is a number): to move relatively from current step
 * string (other formats): to find a step which have the same name
It is also possible to provide a function which should returns one of these types (to manage different navigation depending on states).

It is also possible to set a props "step" to indicate from which step the tutorial should start or if it is already open it navigate directly to this step.

It is available from version 0.13.

I hope it will fit your needs.
